Assets/Scripts/WorldEngine/Groups/CellGroup.cs

```diff
@@ -208,7 +208,7 @@ public float TotalPolityProminenceValue
 #endif
 
     [XmlIgnore]
-    public Dictionary<string, BiomeSurvivalSkill> _biomeSurvivalSkills = new Dictionary<string, BiomeSurvivalSkill>(Biome.Biomes.Count);
+    public Dictionary<string, BiomeSurvivalSkill> _biomeSurvivalSkills = new Dictionary<string, BiomeSurvivalSkill>();
```

Comment:
Good fix to prevent null pointer

Reply:
Yeah. The only reason I did that before was because Biomes where hardcoded and I though I was being efficient by initializing dictionaries to their maximum expected size. But now that biomes are moddable and the Biome collection can grow in size, this "optimization" is moot (and buggy).